README.md

```diff
@@ -150,7 +296,7 @@ To use OpenTok's framework you need a Session ID, a Token, and an API Key. You c
 
 ## Accelerator Core as a dependency
 
-The Accelerator Core is required whenever you use any of the OpenTok accelerators. The Accelerator Core is a common layer that includes the audio-video communication logic contained in all [OpenTok One-to-One Communication Sample Apps](https://github.com/opentok/one-to-one-sample-apps), and permits all accelerators and samples to share the same OpenTok session. The accelerator packs and sample app access the OpenTok session through the Common Accelerator Session Pack layer, which allows them to share a single OpenTok session:
+The Accelerator Core includes the audio-video communication logic contained in all [OpenTok One-to-One Communication Sample Apps](https://github.com/opentok/one-to-one-sample-apps). The accelerator and sample app access the OpenTok session through the Accelerator Session Pack layer, which allows them to share a single OpenTok session:
```

Comment:
I think https://github.com/opentok/one-to-one-sample-apps is deprecated. So maybe, we could skip it